File: font_end/fruitsapp_admin/src/Models/Role.js
class Role {
    constructor(id, role_code, role_name, create_at, update_at, disable) {
        this.id = id;
        this.role_code = role_code || null;
        this.role_name = role_name || null;
        this.create_at = create_at;
        this.update_at = update_at;
        this.disable = disable;
    }
}

export default Role;

File: back_end/fruitsapp_backend/Models/Role.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace fruitsapp_backend.Models
{
	public class Role
	{
       

        [Key]
        public int Id { set; get; }
        public string? role_code { set; get; }
        public string? role_name { set; get; }
        public string? description { set; get; }
        public DateTime create_at { set; get; }
        public DateTime update_at { set; get; }
        public bool isDelete { set; get; }
    }
}

